CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
+  * [Windows Server 2025](https://learn.microsoft.com/azure/aks/windows-node-pools) support is now Generally Available for AKS clusters. Containerd 2.0 will be default with Windows Server 2025.
+  * [Kubelet Serving Certificate Rotation (KSCR)](aka.ms/aks/kubelet-serving-certificate-rotation) is now enabled by default in further regions. Existing node pools will have KSCR enabled by default when they perform their first upgrade to any Kubernetes version 1.27 or greater, while new node pools on Kubernetes version 1.27 or greater will have KSCR enabled by default.
+  * Added support for the service.beta.kubernetes.io/azure-load-balancer-tcp-idle-timeout annotation for Istio ingress gateways.
+  * Kubernetes Event-Driven Autoscaling (KEDA) is now supported in LTS.
+  * [Component Version API](https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster) is available for retrieving the exact versions of all AKS components in your cluster. This gives you real-time visibility into AKS components and add-ons, helping with proactive upgrade planning. Comparing component versions in your current version (cluster context) with potential K8s minor version updates to help identify possible breaking changes. CLI extension 18.0.0b19+ is required.
+
+* Preview Features
+  * [Azure Virtual Network Verifier](https://learn.microsoft.com/en-us/azure/virtual-network-manager/concept-virtual-network-verifier) is now available in Azure Portal (Nodepools blade) for troubleshooting outbound connectivity issues in your AKS cluster. 
+  * [Encryption in transit](https://learn.microsoft.com/en-us/azure/storage/files/encryption-in-transit-for-nfs-shares?tabs=azure-portal%2CUbuntu) is now available for the Azure File CSI driver for AKS version 1.33.
+  * Node Autoprovisioning metrics are now available through Managed Prometheus.
+  * Blue-green nodepool upgrade is now available in Preview in the v20250702preview API.
+ 
+* Bug Fixes
+  * Fixed [issue](https://github.com/Azure/AKS/issues/4720) where AKS evicted pods that had already been manually relocated, causing upgrade failures. This fix adds a node consistency check to ensure the pod is still on the original node before retrying eviction.
+  * Fixed an issue with image pull secrets settings when embargo CVE toggle is enabled.
+
+* Behavior Changes
+  * The delete-machines API will only delete machines from the system nodepool if the system addon PDBs are respected.
+  * Application routing component Pods are now annotated with kubernetes.azure.com/set-kube-service-host-fqdn to automatically have the [API server's domain name injected into the pod](https://learn.microsoft.com/azure/aks/outbound-rules-control-egress#required-outbound-network-rules-and-fqdns-for-aks-clusters) instead of the cluster IP, to enable communication to the API server. This is useful in cases where the cluster egress is via a layer 7 firewall.
+  * Add node anti-affinity for FIPS-enabled nodes for retina-agent when pod-level metrics are enabled.
```

Comment:
component update i think FIPS enablement.

Reply:
Also put the link pls

Reply:
Moved to component update.